src/components/App/styled.ts

```diff
@@ -0,0 +1,14 @@
+import { createGlobalStyle } from 'styled-components'
+
+export const GlobalStyles = createGlobalStyle`
+	body {
+		margin: 0;
+		font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto',
```

Comment:
Why are we adding all the font families? :sweat_smile:

Reply:
They had been already there in `index.css`, I just moved them here, because of usage of styled-components. :sweat_smile: